packages/duckdb-wasm/test/nodefs.test.ts

```diff
@@ -0,0 +1,71 @@
+import * as duckdb from '../src/';
+import { tmpdir } from 'os';
+import { randomUUID } from 'crypto';
+import path from 'path';
+import { unlink } from 'fs/promises';
+
+export function testNodeFS(db: () => duckdb.AsyncDuckDB): void {
+    const files: string[] = [];
+
+    afterAll(async () => {
+        await Promise.all(files.map(file => unlink(file).catch(() => {})));
+        await db().flushFiles();
+        await db().dropFiles();
+    });
+
+    describe('Node FS', () => {
+        it('Should not create an empty DB file in read-only mode for non-existent path', async () => {
+            const tmp = tmpdir();
```

Comment:
Usage of `tmpdir()` seems to not work in the CI test setup, could you have a look at the failure?

Reply:
That's okay. I was also trying to run tests in my local environment with some of the intermediate artifacts produced by the CI to reduce the build time, but it didn't run smoothly as the CI did 😿